lib/Controls.tsx

```diff
@@ -138,8 +273,22 @@ export const Controls: React.FC<ControlsProps> = ({
             />
           </VolumeContainer>
         ) : null}
-        <Progress />
-        <Time />
+        <Progress>
+          <ProgressBar>
+            <ProgressBarPlayed fraction={progress.playedFraction} />
+            <ProgressBarBuffered fraction={progress.bufferedFraction} />
+            <ProgressBarSeek onClick={seek} />
+          </ProgressBar>
+          <ProgressTimestamp>
+            {progress.duration === Infinity
+              ? '∙ LIVE'
+              : `${Duration.fromMillis(
```

Comment:
Could you please spend 5 minutes googling how this could be solved without importing Luxon? If it's too complex I'm fine with this solution.

Reply:
I don't really need to google that, it would be something along the lines of (if `x` is total seconds)
```
h = Math.floor(x/3600)
m = Math.floor((x-h*3600)/60)
s = Math.floor(x-h*3600-m*60)
```
and then put it together in a string.
However, I'm considering support for actual date and time (with locale support), in which case I would need luxon anyway.

Reply:
Show off... :wink: 

In that case I have no problem with using Luxon.